stable/filebeat/values.yaml

```diff
@@ -1,15 +1,10 @@
 image:
   repository: docker.elastic.co/beats/filebeat-oss
-  tag: 6.7.0
+  tag: 7.0.1
   pullPolicy: IfNotPresent
 
 config:
   filebeat.config:
-    prospectors:
```

Comment:
so now that this is removed do inputs get loaded automatically and we don't need this info?

Reply:
Yes, I have also tested it.